apps/fishing-map/features/workspace/highlight-panel/highlight-panel.content.ts

```diff
@@ -58,8 +71,7 @@ const HIGHLIGHT_CONFIGS: HighlightPanelConfig[] = [
   },
   {
     dataviewInstanceId: 'vms-with-png',
-    // TODO: Set to a proper fixed date when release date is defined.
-    releaseDateTimestamp: NEXT_YEAR,
+    releaseDateTimestamp: Date.parse('2023-06-01T00:00:00Z'),
```

Comment:
what about having a const to share the value here and [here](https://github.com/GlobalFishingWatch/frontend/pull/2070/files#diff-aee046b5bdad52f76d3101c7874e8b145c0efec9c128b12115492a37939414f6R73)

Reply:
Great idea @j8seangel , it makes total sense! Take a look at 28c9658ec .